binding.gyp

```diff
@@ -20,7 +20,7 @@
     "xcode_settings": {
       "MACOSX_DEPLOYMENT_TARGET": "10.9",
```

Comment:
(not a real reviewer but) I wonder if for internal consistency's sake this should be bumped like so? Based on what I found it seems like folks on 10.9 _may_ not have a clang that supports C++14, but 10.10+ folks surely will.

```suggestion
      "MACOSX_DEPLOYMENT_TARGET": "10.10",
```

References:

- Clang got full C++14 support in version 3.4 ([source](https://clang.llvm.org/cxx_status.html#cxx14))
- Xcode got Clang 3.4 in version 5.1 ([source](https://gist.github.com/yamaya/2924292#file-xcode-clang-vers-L66-L69)), which was released around [March 2014](https://9to5mac.com/2014/03/10/xcode-5-1-iad-producer-4-1-2-released-with-improved-ios-7-1-compatibility/)
- Xcode 5.1's [release notes](https://developer.apple.com/library/archive/documentation/Xcode/Conceptual/WhatsNewXcode-Archive/Articles/xcode_5_0.html#//apple_ref/doc/uid/TP40012953-SW31) state "Xcode 5.1 requires a Mac running OS X 10.8.4 (or later), or OS X 10.9."
- macOS 10.10 was the first version of OS X released following the release of Xcode 5.1